### DISCUSSION TITLE
Windows Server 2019 Support

### DISCUSSION THREAD

**Discussion by bhonushriram** [farag2/Sophia-Script-for-Windows#109] [Q&A] [answered]
Can you please guide me how I can add windows server 2019 support in Sophia script?

Thank you

**Comment by farag2** [comment, marked as answer]
Hello. You can remove this part

https://github.com/farag2/Windows-10-Sophia-Script/blob/349a46ff62015f1d9be27406370a50cacc9b9b22/Sophia/Sophia.psm1#L53

or add the right BuildNumber be getting from Windows Server 2019 (I haven't got one to test). But I do not know which functions will work or not on that kind of OS. Do it on your risk.
